src/components/ResultsView.tsx

```diff
@@ -441,13 +498,62 @@ const ResultsView: ResultsViewComponent = ({
                             value: l.id,
                             blockUid: layoutNode.uid,
                           });
+                          setIsEditLayout(false);
+                          setMoreMenuOpen(false);
                         }}
                       >
                         <Icon icon={l.icon} />
                         <span className="capitalize text-sm">{l.id}</span>
                       </div>
                     ))}
                   </div>
+                  {settingsById[layoutMode].map(
+                    (s) =>
+                      s.options.length === 1 && (
+                        <Label key={s.key}>
+                          {s.label}
+                          <Checkbox
+                            defaultChecked={layout.style === s.options[0]}
+                            label={s.options[0]}
+                            onChange={(e) => {
+                              const resultNode = getSubTree({
+                                key: "results",
+                                parentUid,
+                              });
+                              const layoutNode = getSubTree({
+                                key: "layout",
+                                parentUid: resultNode.uid,
+                              });
+                              setInputSetting({
+                                key: "mode",
+                                value: layoutMode,
+                                blockUid: layoutNode.uid,
+                              });
+                              setLayout({
+                                ...layout,
+                                [s.key]: (e.target as HTMLInputElement).checked
+                                  ? s.options[0]
+                                  : "default",
+                              });
+                              setInputSetting({
+                                key: s.key,
+                                value: (e.target as HTMLInputElement).checked
+                                  ? s.options[0]
+                                  : "default",
+                                blockUid: layoutNode.uid,
+                              });
+                              setShowInterface(
+                                layout.style !== "Minimal" &&
```

Comment:
Not a big fan of this

Reply:
The conflict here appears to be doing something that feels like it's layout specific but is actually resultsview-wide.

I think a decision should be made here: should a interface toggle be layout specific or global? then commit to the decision

Reply:
I went with global